CONTRIBUTING.md

```diff
@@ -52,6 +52,8 @@ and is repeatable in any way, should be tested, to guarantee backwards compatibl
 - try not to introduce lines longer than 100 characters
 - write small functions whenever possible, and use descriptive names for functions and variables
 - comment tricky or non-obvious decisions made to explain their rationale
+- the tool is linted with `shellcheck` and formatted with `shfmt`: make sure you run these locally
+to avoid CI issues (their options are listed in `lint.yml`)
```

Comment:
Lemme check. I think it's just GitHub acting up 😄

Reply:
I see it like this, newline after the word `locally`

<img width="807" alt="image" src="https://github.com/kerl/kerl/assets/7303768/83b385a8-69b2-4890-9956-173679d60a28">

or did you mean something else?

Reply:
In any case, since I got your approval, I'll go ahead and squash-merge to push more goodies. Feel free to reply to this and I'll act on it if there's something to change 😄 
Thanks.

Reply:
Nothing to change, just wanted to make sure you wanted the text that way